packages.json

```diff
@@ -20513,5 +20513,20 @@
     "description": "Create ICS files for email invites, eg. invite.ics",
     "license": "MIT",
     "web": "https://github.com/ThomasTJdev/nimics"
+  },
+  {
+    "name": "colorizeEcho",
+    "url": "https://github.com/s3pt3mb3r/colorizeNim",
```

Comment:
Shouldn't this be `https://github.com/s3pt3mb3r/colorizeEcho`? (Yes, your link gets redirected, but maybe some day it won't)

Reply:
Oh, yes it must be colorizeEcho! It's because I changed repo name midway.
Could you change it to `ttps://github.com/s3pt3mb3r/colorizeEcho` or should I modify somehow? I'm sorry I'm not familiar to PR so...

Reply:
```suggestion
    "url": "https://github.com/s3pt3mb3r/colorizeEcho",
```